CONTRIBUTING.md

```diff
@@ -66,97 +67,32 @@ clients that need to extract a single description/example.
 
 ## Submitting a pull request
 
-TL;DR: fork, clone, `npm install`, feature branch, commit, push, pull request, check Travis.
+For submitting changes, you can use whatever workflow you're more comfortable with.
+The easiest way is to just edit the page directly on the Github interface.
+Check out the step-by-step instructions (with screenshots) on
+[Github Help](https://help.github.com/articles/editing-files-in-another-user-s-repository/).
 
-Detailed explanation:
+Alternatively, you can do it using the command line:
```

Comment:
Yes, I thought about that. Even though for a newcomer the web-based workflow would be easier, it fits the spirit of this project to help people learn to use tools that may seem daunting at first :)

I'm thinking of doing something like this: https://www.mediawiki.org/wiki/Gerrit/Getting_started (simplified wherever possible/relevant). What do you think?